### PR TITLE
0.4.1 releasenotes

### PR DESCRIPTION
**Pull request by j-wags**
Docs branch built at: https://open-forcefield-toolkit.readthedocs.io/en/041-releasenotes/releasehistory.html

- [x] #371 merged
- [x] Update [changelog](https://github.com/openforcefield/openforcefield/blob/master/docs/releasehistory.rst)

### CODE REVIEW

File: docs/releasehistory.rst [hunk 1]
```diff
@@ -11,25 +11,72 @@ Releases follow the ``major.minor.micro`` scheme recommended by `PEP440 <https:/
 Current development
```
Comment (jchodera) [inline docs/releasehistory.rst]: Should this be 0.4.1?

File: docs/releasehistory.rst [hunk 2]
```diff
@@ -11,25 +11,72 @@ Releases follow the ``major.minor.micro`` scheme recommended by `PEP440 <https:/
...
 -------------------
```
Comment (jchodera) [inline docs/releasehistory.rst]: Do you need release summary text here?

**Comment by j-wags** [comment]
@jchodera I've addressed your points above and make a few other fixes. Please take another look when possible. If it looks good I'll take care of the rest of the release.

**Comment by j-wags** [comment]
Thanks -- The version number and description were total omissions on my part. I need to update the release checklist with this stuff -- Both to keep me on track when there's a lot going on, and to improve my bus factor :-)

**Pull request merged**